.github/workflows/release.yml

```
@@ -30,7 +30,22 @@ jobs:
 
       - name: Run tests
         run: deno run test
-        
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)